### PR TITLE
doc: enable auto gh-pages generation in docker script

### PR DESCRIPTION
**Pull request by lukaszstolarczuk**
I've added this on stable-1.8 but the question is - do we want to have it for all branches?

---
This change is [Reviewable](https://reviewable.io/reviews/pmem/libpmemobj-cpp/656)

### CODE REVIEW

**Comment by lukaszstolarczuk** [comment]
@marcinslusarz, could you please take a look if this makes sense to put all current (valid_)branches in PR for stable-1.5 ?

File: utils/docker/run-doc-update.sh [hunk 1]
```diff
@@ -32,20 +32,33 @@
...
+
+BOT_NAME="pmem-bot"
+USER_NAME="pmem"
+REPO_NAME="libpmemobj-cpp"
+
+ORIGIN="https://${GITHUB_TOKEN}@github.com/${BOT_NAME}/${REPO_NAME}"
+UPSTREAM="https://github.com/pmem/${REPO_NAME}"
+# master or stable-* branch
+TARGET_BRANCH=${TRAVIS_BRANCH}
+VERSION=${TARGET_BRANCHES[$TARGET_BRANCH]}
```
Comment (marcinslusarz) [inline utils/docker/run-doc-update.sh]: this will end badly when TRAVIS_BRANCH is empty (eg on gha), see my recent fix to pmdk

**Comment by marcinslusarz** [comment]
> @marcinslusarz, could you please take a look if this makes sense to put all current (valid_)branches in PR for stable-1.5 ?

it looks funny, but there's no harm in doing it this way, so lgtm

File: utils/docker/run-doc-update.sh (hunk 1 above)
Comment (lukaszstolarczuk) [reply utils/docker/run-doc-update.sh]: Done.

**Comment by lukaszstolarczuk** [comment]
@marcinslusarz , thanks and done.

File: utils/docker/run-doc-update.sh (hunk 1 above)
Comment (marcinslusarz) [reply utils/docker/run-doc-update.sh]: This still doesn't look correct.

https://github.com/pmem/pmdk/commit/eeb8fd86c6c72f4607a22a1e9d9f7da31be8af2a

**Pull request merged**